app/views/devise/registrations/edit.html.erb

```diff
@@ -31,6 +31,11 @@
     <%= f.password_field :current_password, autocomplete: "current-password" %>
   </div>
 
+  <div class="field">
+    <%= f.label :phone_number %><br />
+    <%= f.phone_field :phone, autocomplete: "phone" , placeholder: "+(91)123456789"%>
```

Comment:
while updating, no need to give placeholder for the phone number, you can display user's phone number if you want to.